rs/orchestrator/src/registration.rs

```diff
@@ -66,6 +66,7 @@ pub(crate) struct NodeRegistration {
     key_handler: Arc<dyn NodeRegistrationCrypto>,
     local_store: Arc<dyn LocalStore>,
     signer: Box<dyn Signer>,
+    node_reward_type: Option<String>,
```

Comment:
Is this field needed? It seems unused

Reply:
You mean unused on the NNS side? Yes at the moment, but we plan to adjust rewards based on this field.
Or you mean elsewhere?

Reply:
I mean here in the orchestrator, because blow in `assemble_add_node_message` we take the value from the `node_config` directly

Reply:
Ah, indeed, good point. Thanks, removed the field from here!